src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCAUNT = 50;
```

Comment:
There is a typographical error in the constant name 'LONG_TERM_DISCAUNT'. It should be 'LONG_TERM_DISCOUNT'.